_data/tasks.yml

```diff
@@ -23,4 +23,7 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
-
+- title: Stateful Set
+  section:
+  - title: Upgrade from PetSet to Stateful Sets
```

Comment:
I think it sounds better to keep everything singular (`s/sets/set/ig`) and consistent (e.g. the title says `PetSet` but the URL below is `petsets`).